src/tree.rs

```diff
@@ -264,6 +264,8 @@ impl<Pane> Tree<Pane> {
         }
 
         self.preview_dragged_tile(behavior, &drop_context, ui);
+        // resizes the window to fit the ui.
+        ui.allocate_space(ui.available_size());
```

Comment:
We use `ui.available_rect_before_wrap()` earlier.

Save that rect instead, then use `ui.advance_cursor_after_rect(rect);` here

Reply:
ah, didn't notice that. changed pr to re-use rect.